src/_pytest/compat.py

```diff
@@ -381,3 +381,14 @@ def __get__(self, instance, owner=None):  # noqa: F811
                 return self
             value = instance.__dict__[self.func.__name__] = self.func(instance)
             return value
+
+
+# Sometimes an alogirthm needs an order-preserving dict. Since Python 3.7,
```

Comment:
```suggestion
# Sometimes an algorithm needs an order-preserving dict. Since Python 3.7,
```

Reply:
Thanks; I also clarified the comment a bit.